src/custom/utils/analytics/otherEvents.ts

```diff
@@ -7,3 +7,25 @@ export function gameAnalytics(gameType: GameType) {
     action: `Playing ${gameType} game`,
   })
 }
+
+function _detectExplorer(href: string) {
```

Comment:
don't we have some util already for this that takes the chainId? do we not have access to chainId or sth?

Reply:
Maybe in here you get some reusable pieces https://github.com/cowprotocol/cowswap/blob/5b8bf3e4f176d452433a0118dcb3caa65ffebdb5/src/custom/utils/index.ts#L64

which btw, I think is weird that we have all that in the `index.ts` and we should move it to `utils/explorers.ts`

Anyways, what about Etherscan?